src/refer.ts

```diff
@@ -35,39 +33,61 @@ function refer(this: any, options: any) {
     return {
       ok: true,
       entry,
-      occur: [occur]
+      occur: [occur],
     }
   }
 
+  async function actAcceptEntry(this: any, msg: any) {
+    const seneca = this
+
+    const entry = await seneca.entity('refer/entry').load$({ key: msg.key })
+
+    if (!entry) {
+      return {
+        ok: false,
+        error: 'No entry found with this key',
```

Comment:
use the convention:
```
{ ok:false, why:'entry-unknown', details: { ... } }
```
* why should a "code", not human text
* details are optional
*

Reply:
Got it! Return updated.